ote_sdk/ote_sdk/usecases/exportable_code/demo/README.md

```diff
@@ -16,15 +16,23 @@ Demo package contains simple demo to get and visualize result of model inference
 
 ## Prerequisites
 * Python 3.8+
+* Git
 
 ## Setup Demo Package
 
-1. Install Python (version 3.8 or higher), [setuptools](https://pypi.org/project/setuptools/), [wheel](https://pypi.org/project/wheel/).
+1. Install Python (version 3.8 or higher).
 
 2. Install the package in the clean environment:
-```
-python -m pip install demo_package-0.0-py3-none-any.whl
-```
+
+   Please make sure that the environment contains [wheel](https://pypi.org/project/wheel/) and if not, please call the following command:
```

Comment:
can we just include wheel into requirements.txt?

Reply:
No, we can't. wheel is needed only for installation step and if we add `wheel` in reqs.txt, but virtual env will not contains `wheel`, we get error. I added description for step about creation virtual env and if the user follows the instructions, the installation stage will be successful. Anyway I think it is better if README contains info about `wheel` (in cases when user use `venv`).

Reply:
ok, then